src/bundle/lookup.js

```diff
@@ -0,0 +1,42 @@
+define([
+	"../core/likely_subtags",
+	"../core/remove_likely_subtags",
+	"../core/subtags",
+	"../util/array/for_each"
+], function( coreLikelySubtags, coreRemoveLikelySubtags, coreSubtags, arrayForEach ) {
+
+	/**
+	 * bundleLookup( minLanguageId )
+	 *
+	 * @Cldr [Cldr class]
+	 *
+	 * @cldr [Cldr instance]
+	 *
+	 * @minLanguageId [String] requested languageId after applied remove likely subtags.
+	 *
+	 * Ref: http://goo.gl/qpwZ4n
+	 */
+	return function( Cldr, cldr, minLanguageId ) {
```

Comment:
When the only difference is the case of a single character, you have bad variable names.